test/application/modules/legacy/commands/register_keys.spec.ts

```
@@ -64,7 +65,7 @@ describe('Register keys command', () => {
 
 	describe('constructor', () => {
 		it('should have valid name', () => {
-			expect(registerKeysCommand.name).toBe(COMMAND_NAME_REGISTER_KEYS);
+			expect(registerKeysCommand.name).toBe(COMMAND_REGISTER_KEYS);
```

Please update the test case description from `should throw error when validator has a no BLS keys` to `should throw an error when a validator has no BLS key`